### ISSUE TITLE
Boilerplate from one openapi spec

### ISSUE THREAD

**Issue by dwebfan** [contiamo/restful-react#355]
**Is your feature request related to a problem? Please describe.**
Thanks for the project helping people to auto-generate most codes easily. But it would be great to also generate the whole project as boilerplate if possible.

**Describe the solution you'd like**
Similiar as https://github.com/pankod/next-boilerplate, if people can run cli by providing one openAPI spec, one project can be autocreated, even some simple layout can be created, it would be super useful for any new people.

**Describe alternatives you've considered**

**Additional context**

**Comment by dwebfan** [comment]
yes, #1 and #2 is exactly what I mean. I feel superplate has bootstrap one project capability but lack of openAPI support, and also mainly focus node.js backend, not open for other language. But your project can leverage openAPI spec and auto-create UI sides codes, so I feel they are complementary, and the integration btw this two projects would be very powerful.

**Comment by fabien0102** [comment]
I like the idea, if you want to play with a superplate plugin, please do it and we can add the plugin in the `README.md`

For the `restful-react init` this is my todolist since a long time, I'm just missing time for side projects 😅
